features/proposed/check-domain-mx/README.md

```diff
@@ -0,0 +1,71 @@
+
+Check Domain MX
+===============
+
+As a user signing up for Firefox Accounts,
+I want to receive a clear error message
+when I mistype my email address,
+and the typo'd domain is
+obviously not capable of receiving mail.
+
+## UX Mocks
+
+<img src="no-mx-error.png">
+
+## Success Criteria
+
+We will measure success
+by measuring the rate of email bounces
+(`fxa.auth.account.email_bounced`)
+as a percentage of email verifications
+(`fxa.auth.account.verified`).
```

Comment:
You caught me; there was some weirdness with the `fxa.auth.account.created` events in datadog that I couldn't figure out, so I punted.  It is indeed a better denominator so we should use it.